src/python/grpcio_tests/tests/reflection/_reflection_servicer_test.py

```diff
@@ -50,6 +50,16 @@ def _file_descriptor_to_proto(descriptor):
 
 class ReflectionServicerTest(unittest.TestCase):
 
+    # NOTE(lidiz) Bazel + Python 3 will result in creating two different
+    # instance of DESCRIPTOR for each message. So, the equal comparision
+    # between protobuf returned by stub and manually crafted protobuf will
+    # always fail.
+    def _assert_sequence_of_proto_equal(self, x, y):
+        self.assertSequenceEqual(
+            list(map(lambda x: x.SerializeToString(), x)),
```

Comment:
Do you think we should file a bug against the python protobuf compiler for this?

Reply:
I attempt several times to make a minimum reproducible example without gRPC package involved, but no luck yet.

Reply:
This test appears to pass for me locally, without this change. Is bazel+python3 run as part of our CI with this PR?

Reply:
It disappears after I do `bazel clean --expunge`. I have reverted this hack.

It maybe introduced by mixing the artifacts of Python 2 and Python 3 on my local machine... Further more, since I added `force_python=PY3`, the generated artifacts in latest version of this PR will not be mixed but separated in different folders. I presume it might not happen again...

Reply:
Also, I enabled Python 3 Bazel test for future PRs.

Reply:
From discussion offline this override here is still necessary. Please file an issue and change the `NOTE(lidiz)` to `TODO(issue link)`.  If I remember correctly we can't reproduce without gRPC involved, so the issue should just go on our repo for now.